addons/sourcemod/scripting/SurfTimer.sp

```diff
@@ -1900,7 +1925,7 @@ public void OnMapStart()
 
 	// Save Locs
 	ResetSaveLocs();
-
+	g_iallowCheckpointRecreation = GetConVarInt(g_hallowCheckpointRecreation);
```

Comment:
this should go to convars.sp on line 371

Reply:
Done